src/Button/Button.tsx

```diff
@@ -6,16 +6,23 @@ import '../../lib/design-tokens/css/tokens/functional/components/button/colors-w
 import type {BaseProps} from '../component-helpers'
 import styles from './Button.module.css'
 
+export enum LabelFontSize {
```

Comment:
What was the reason behind using an enum here, rather than relying on the union like other types. Also noticed that it's being exported from this file, so it'll end up in the package root. I'm not sure that's ideal 🤔. Can we stick to the union?

Reply:
I have no issue removing the export if we don't want external access to this

Reply:
👍 yeah, let's not surface the object in the main index.

Ash has confirmed there's no sizes besides `medium` and `large` if that affects the decision around doing it via ternary instead of enum. Also, as it's going to be a private variable, can we call it `ButtonTextSizeLookup` (or similar) instead, so the name indicates it's a map?